docs/i2c_driver.md

```diff
@@ -211,9 +211,32 @@ Writes to a register on the I2C device.
 
 ---
 
+### `i2c_status_t i2c_writeReg16(uint8_t devaddr, uint16_t regaddr, uint8_t* data, uint16_t length, uint16_t timeout)`
+
+Writes to a 16-bit register on the I2C device.
```

Comment:
Writes to a register with a 16-bit address.

Also should mention that the address is transmitted big-endian.